codes/classical/bits/turbo.yml

```diff
@@ -0,0 +1,41 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+#this entry is far from complete. expect updates in the following days.
+
+code_id: turbo
+physical: bits
+logical: bits
+
+name: 'Turbo code'
+introduced: '\cite{doi:10.1109/ICC.1993.397441}'
+
+description: 'Codes with composite structure, where information bits are encoded with many different component codes. Between encodings, these codes are permuted in an interleaving fashion.'
```

Comment:
This is not precise enough to give me a feel for these codes. Please try a new first sentence + also a more technical defn. See 21.8.1 Huffman Kim Sole.